<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>

<body>

    <script>

        //  If the error is happening or not finally run will always

        // In this code i writed a try catch block in try  block an error is printing which i have catched in catch block and the finally is running always if the error is happening and the error is not happening

        try {
            console.log(anything)
        } catch (err) {
            console.log(err)
            console.log(a)
        }
        // If does not write it in finally block it give me an error
        finally {
            console.log("I always run")
        }

    </script>

</body>

</html>